skfem/assembly/basis/__init__.py

```diff
@@ -10,16 +8,4 @@
 Basis = CellBasis
 InteriorBasis = CellBasis  # backwards compatibility
 ExteriorFacetBasis = BoundaryFacetBasis  # backwards compatibility
-
-
-def FacetBasis(*args, side=None, **kwargs):
-    """alias of :class:`~skfem.assembly.BoundaryFacetBasis`"""
-    if side is None:
-        return BoundaryFacetBasis(*args, **kwargs)
-    warnings.warn("Initializing FacetBasis using the keyword argument side "
-                  "is deprecated. Use InteriorFacetBasis or MortarFacetBasis "
-                  "instead.", DeprecationWarning)
-    if 'mapping' in kwargs:
-        if hasattr(kwargs['mapping'], 'helper_to_orig'):
-            return MortarFacetBasis(*args, side=side, **kwargs)
-    return InteriorFacetBasis(*args, side=side, **kwargs)
```

Comment:
This is something I had before to be super compatible. I think it can be removed now because these features are not used too often.